ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -201,25 +200,32 @@ const DeviceColumn = ({ deviceData, devices, idPrefix, handleRequestChange, lock
     );
 };
 
-const FormatColumn = ({ deviceData, handleRequestChange, idPrefix, request }) => {
+const FormatSwitch = ({ handleRequestChange, idPrefix, request, isRootMountPoint }) => {
```

Comment:
Making in seperate component is an overkill. Just make it a const variable inside the FormatColumn